File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.py
n, x = map(int,input().split())
a = list(map(int,input().split())) # 수열 a (일정한 규칙에 따라 한 줄로 배열된 수의 열.)
for i in a: # Ex) a[1] < x, a[2] < x, a[3] < x ... 반복되며,
    if i < x: # 참일 경우, 출력
        print(i, end=" ") # end=" " : 한 칸 공백

File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.c
#include <stdio.h>

// x 보다 작은 수를 입력받은 순서대로 공백으로 구분해 출력, x보다 작은 수는 적어도 하나가 존재.
// n 번의 입력을 받아 x보다 작은 수를 출력해야 함 !! 입력 받자마자 출력 !!

int main(){
    int N, X, input;

    scanf("%d %d", &N, &X);

    for(int i=0; i<N; i++){
        scanf("%d", &input);
      
        if(X>input)
            printf("%d ", input);
    }
}